cmd/snap/cmd_login.go

```diff
@@ -111,6 +111,8 @@ func (x *cmdLogin) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	fmt.Fprint(Stdout, i18n.G("Authenticated operation on this device will be under terms of service found at: https://www.ubuntu.com/legal/dataprivacy/snap-store")+"\n\n")
```

Comment:
"Personal information is handled as per our privacy notice at https://www.ubuntu.com/legal/dataprivacy/snap-store"

^ has been +1'd by Legal, let's do it!